### DISCUSSION TITLE
getting the cartesian coordinates though distance matrix

### DISCUSSION THREAD

**Discussion by orgw** [rdkit/rdkit#6204] [Q&A]
hi all,

let's say we have a ligand and protein. The atoms of ligand and proteins are in 3 dimensional space, with coordinates and elements. Now, if we have the distance matrix of ligand atoms and the distance matrix between ligand and protein atoms, and also the cartesian coordinates of the protein atoms, is it possible in python to get the cartesian coordinates of the ligand?

thanks

**Comment by hypnopump** [comment]
there are a few tricks one could do. but overall, the situation is not the ideal. could you try to get the coordinates directly?
